assets/css/variables.scss

```diff
@@ -1,9 +1,9 @@
 $red: crimson;
 
 $colorUnknown: #ff767f;
-$colorAvailable: #76bd79;
+$colorAvailable: #24c603;
```

Comment:
We should keep those colors in the planning, we can maybe create new variables for calendar